### ISSUE TITLE
Mini MK3

### ISSUE THREAD

**Issue by simpleas0** [dbkynd/controlcast#53]
Does anyone know if this works with the Novation Launchpad MK3 Mini? I have a MK2 Mini on the way but the MK3 has some nice improvements I'd just go for if it's compatible

**Comment by ruohki** [comment]
I dont think this tool is maintened so i started my own project that already does support the mini mk2, S, MK2 and mini mk3 . No source release yet as I need to shape things but if you want to grab it you can do so in the discord I did setup for that purpose: https://discord.gg/4Ys9TRR
